src/lib.rs

```diff
@@ -1025,6 +1025,26 @@ impl<'a> WebViewBuilder<'a> {
   }
 }
 
+/// Request from a IPC message. See [`WebViewBuilder::with_ipc_handler`] for more information.
+pub struct IpcRequest {
```

Comment:
instead of a new type, can't we just pass the url as a second argument with `Option<String>`?

Reply:
maybe we should use http::Request instead? we might add new parameters to this closure in the future (like a closure to provide a response to the webview) and it's more type safe to have a dedicated type for multiple parameters